### ISSUE TITLE
Using non-interger values in css tranforms for horizontal scroll causes bluriness in Safari on Mac 

### ISSUE THREAD

**Issue by sergek** [schrodinger/fixed-data-table-2#254]
Hello, feeding non-rounded values to translate3d in a translateDOMPositionXY.js file causes the whole transformed area to appear blurred in Safari, Chrome on Mac 

https://github.com/schrodinger/fixed-data-table-2/blob/master/src/vendor_upstream/dom/translateDOMPositionXY.js#L32 

I believe that Chrome still uses the Webkit rendering engine on Mac, because I see this behavior in Chrome also.

I see no blurring if I change the code to just use the integer part of a value like this:
```
return function ( /*object*/style, /*number*/x, /*number*/y) {
        style[TRANSFORM] = 'translate3d(' + parseInt(x, 10) + 'px,' + parseInt(y, 10) + 'px,0)';
        style[BACKFACE_VISIBILITY] = 'hidden';
 };
```
Also I see no positioning errors, but I tested this only on Mac.

What do you guys think - are there any non-obvious drawbacks of such rounding? Any better ways of fixing this?

**Comment by wcjordan** [comment]
Works for me, but since this is pulled from fbjs, could you make a discuss with the fbjs maintainers, and then make a PR here to replace this file with an fbjs dependency?

https://github.com/facebook/fbjs/blob/c69904a511b900266935168223063dd8772dfc40/packages/fbjs/src/dom/translateDOMPositionXY.js

**Comment by sergek** [comment]
I don't believe this will be addressed in fbjs
Fbjs is not meant to be a public library and they aren't seem to be very active over issues or pr's.
After looking at it I would say that fbjs is a dangerous dependency.

**Comment by wcjordan** [comment]
Could you point me to the discussion you had with the fbjs maintainers regarding why they wouldn't want to take the change you're suggesting?

**Comment by sergek** [comment]
Hi, there isn't any discussion per se, my question is here https://github.com/facebook/fbjs/issues/273

**Comment by wcjordan** [comment]
Ah hmm, that's not too helpful.  We could switch to a separate library if you know of any alternatives.  Alternatively I can accept a PR in place, but since I don't have much experience or desire to maintain these utilities we should be very careful with changes to them.

**Comment by sergek** [comment]
I agree that it's best to avoid messing with upstream libs over their maintainers.
Also I doubt that there is any drop-in replacement library available.

I've made a pr to add rounding in the fixedTable code itself, it fixes my issue, but I didn't test it extensively (all browsers etc.)

Please look 
https://github.com/schrodinger/fixed-data-table-2/pull/262

**Comment by pradeepnschrodinger** [comment]
Fixed with the [v1.0.0](https://github.com/schrodinger/fixed-data-table-2/releases/tag/v1.0.0) release.
